sql/hive/benchmarks/InsertIntoHiveTableBenchmark-results.txt

```diff
@@ -0,0 +1,11 @@
+Java HotSpot(TM) 64-Bit Server VM 1.8.0_251-b08 on Mac OS X 10.15.4
+Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
+insert hive table benchmark:              Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
+------------------------------------------------------------------------------------------------------------------------
+INSERT INTO DYNAMIC                                7346           7470         175          0.0      717423.0       1.0X
+INSERT INTO HYBRID                                 1179           1188          13          0.0      115184.2       6.2X
+INSERT INTO STATIC                                  344            367          48          0.0       33585.1      21.4X
+INSERT OVERWRITE DYNAMIC                           7656           7714          82          0.0      747622.7       1.0X
+INSERT OVERWRITE HYBRID                            1179           1183           6          0.0      115163.3       6.2X
+INSERT OVERWRITE STATIC                             400            408          10          0.0       39014.2      18.4X
```

Comment:
do you have the numbers before your PR?

Reply:
Let me run this benchmark on the master branch and update the result later in the PR description.

Reply:
```sql
-INSERT INTO DYNAMIC                                7742           7918         248          0.0      756044.0       1.0X
-INSERT INTO HYBRID                                 1289           1307          26          0.0      125866.3       6.0X
-INSERT INTO STATIC                                  371            393          38          0.0       36219.4      20.9X
-INSERT OVERWRITE DYNAMIC                           8456           8554         138          0.0      825790.3       0.9X
-INSERT OVERWRITE HYBRID                            1303           1311          12          0.0      127198.4       5.9X
-INSERT OVERWRITE STATIC                             434            447          13          0.0       42373.8      17.8X
+INSERT INTO DYNAMIC                                7382           7456         105          0.0      720904.8       1.0X
+INSERT INTO HYBRID                                 1128           1129           1          0.0      110169.4       6.5X
+INSERT INTO STATIC                                  349            370          39          0.0       34095.4      21.1X
+INSERT OVERWRITE DYNAMIC                           8149           8362         301          0.0      795821.8       0.9X
+INSERT OVERWRITE HYBRID                            1317           1318           2          0.0      128616.7       5.6X
+INSERT OVERWRITE STATIC                             387            408          37          0.0       37804.1      19.1X
```

+ for master
- for this PR
both using hive 2.3.7